Remotion/Web/Development.WebTesting/BrowserLog/BrowserLogUtility.cs

```diff
@@ -68,27 +72,57 @@ public static IReadOnlyCollection<Regex> GetBrowserLogIgnoredEntryRegexes (ITest
   /// </summary>
   public static IReadOnlyCollection<BrowserLogEntry> GetUnexpectedBrowserLogEntries (
       [NotNull] IBrowserSession session,
+      [NotNull] IBrowserConfiguration configuration,
       LogLevel minLogLevel,
       [NotNull] IReadOnlyCollection<Regex> messageFilter)
   {
     ArgumentUtility.CheckNotNull(nameof(session), session);
+    ArgumentUtility.CheckNotNull(nameof(configuration), configuration);
     ArgumentUtility.CheckNotNull(nameof(messageFilter), messageFilter);
 
-    var remainingRetries = 50;
-    IReadOnlyCollection<BrowserLogEntry> logEntries;
-    do
+    IReadOnlyCollection<BrowserLogEntry> browserLogs;
+    if (configuration.UseBidiLog())
```

Comment:
Wouldn't it be better to test IBrowserSession.Window. We can softcast if the Window is also a BrowserSession (it always is) and then use Window.Browser.UseBidiLog(). that way, we don't need the extra parameter.